kubernetes/celestia-local/deployment.yml

```diff
@@ -69,15 +70,39 @@ spec:
                 name: celestia-env
           ports:
             - containerPort: 26659
-              name: node-svc
+              name: bridge-rest
```

Comment:
we can probably stop exposing this port, see other comments on gateway.

Reply:
We're still calling the rest api in `celestia-local/deployment.yml` for celestia-bridge's `startupProbe`